server/index.js

```diff
@@ -53,6 +53,96 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  if (!req.session.cartId) {
+    res.status(200).json([]);
```

Comment:
Don't forget to `return` here. Responding does not end the callback function and the rest of the code will attempt to run.